Essentials/src/com/earth2me/essentials/UserData.java

```diff
@@ -446,6 +446,22 @@ public void setTeleportEnabled(boolean set) {
         config.save();
     }
 
+    private boolean autoTeleportEnabled;
+
+    private boolean _getAutoTeleportEnabled() {
+        return config.getBoolean("autoteleportenabled", false);
```

Comment:
I'd use a different config key (though I appreciate this is based on `teleportenabled` as used above) - perhaps `"teleportauto"`?

Reply:
Sounds good :+1: